lib/Dialect/CGGI/IR/CGGIOps.td

```diff
@@ -57,6 +59,17 @@ def CGGI_NotOp : CGGI_Op<"not", [
   let summary = "Logical NOT of two ciphertexts";
 }
 
+def CGGI_PackedOp : CGGI_Op<"packed", [
+    Pure]> {
+  let arguments = (ins
+    CGGIGate_Attr:$gates,
```

Comment:
could this op use a verifier to validate that the gate attr is the same length as the lhs/rhs/output? (I think lhs/rhs/output sizes will be verified if you add SameOperandsAndResultType)